### ISSUE TITLE
"clairvoyance" is autofilling for end if, not just if. 

### ISSUE THREAD

**Issue by slylabs13** [trevordevore/livecode-sublimetext#13]
I cannot post a quicktime movie of what is happening. I paid closer attention this time, and what is happening is that when typing END IF and then return, the autofill feature is populating the script as though I had typed IF and then select IF from the autofill popup menu. 

Not sure if this is a bug, but it came up while I was typing a handler like I would normally do. Maybe adding an entry for END IF with no autofill would do the trick?

**Comment by trevordevore** [comment]
Check out [licecap](http://www.cockos.com/licecap/) for creating animated gifs that you can add to reports on GitHub.

**Comment by slylabs13** [comment]
![linter bug recording](https://cloud.githubusercontent.com/assets/14981684/24128662/8e18db24-0d9a-11e7-9d95-81d3f31322f5.gif)

**Comment by trevordevore** [comment]
Thanks for the gif. It looks like you are pressing RETURN after typing "end if". Is that correct? If so then it is the autocomplete is kicking in. Let me know if you are in fact typing RETURN.

**Comment by slylabs13** [comment]
Yes, just like I would when typing a script. I wonder now that you mention it if there is a way to set the default auto-return key.

But still, the autofill is NOT popping up for IF but is popping up for END IF.

Bob S


On Mar 22, 2017, at 07:33 , Trevor DeVore <notifications@github.com<mailto:notifications@github.com>> wrote:

Thanks for the gif. It looks like you are pressing RETURN after typing "end if". Is that correct? If so then it is the autocomplete is kicking in. Let me know if you are in fact typing RETURN.

**Comment by slylabs13** [comment]
I see why IF won't pop up a menu. The number of characters typed does not meet the threshold. IF that is the case, then IF should not be in the autocomplete list.

Bob S


On Mar 22, 2017, at 08:06 , Bob Sneidar <bobsneidar@iotecdigital.com<mailto:bobsneidar@iotecdigital.com>> wrote:

Yes, just like I would when typing a script. I wonder now that you mention it if there is a way to set the default auto-return key.

But still, the autofill is NOT popping up for IF but is popping up for END IF.

Bob S

**Comment by slylabs13** [comment]
NVM. Now IF seems to pop up the menu.

Bob S


On Mar 22, 2017, at 08:35 , Bob Sneidar <bobsneidar@iotecdigital.com<mailto:bobsneidar@iotecdigital.com>> wrote:

I see why IF won't pop up a menu. The number of characters typed does not meet the threshold. IF that is the case, then IF should not be in the autocomplete list.

Bob S

**Comment by trevordevore** [comment]
v0.2.1 fixes this. I changed the auto complete triggers for if, if else, and else if. The old tabTrigger's for these snippets would trigger in cases where they shouldn't. New triggers won't have this problem. 

New triggers:
ifthen
elseif
ifelse

I also added some configuration setting suggestions to the README. I don't like auto complete displaying suggestions while I type. Rather, I like to use Control + Spacebar to show suggestions or type the TAB key to fill in the first suggestion (e.g. auto complete variable names I've declared elsewhere in the file).
